samples/errors8.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
My suggestions summarised here: https://goplay.space/#ISKTAUFOyjk

removed unnecessary blank line, error is not handler, shortened dns error, removed unreachable clause in switch.

Reply:
Thanks used it.